commands/files/serialfile.go

```diff
@@ -69,7 +69,7 @@ func (f *serialFile) NextFile() (File, error) {
 	f.files = f.files[1:]
 
 	// open the next file
-	fileName := fp.Join(f.name, stat.Name())
+	fileName := fp.ToSlash(fp.Join(f.name, stat.Name()))
```

Comment:
out of scope for this PR (probably) but I dont like this being `fp` here and `filepath` everywhere else